assets/css/custom.css

```diff
@@ -549,6 +549,18 @@ li > a {  /* 0 0 0 2 */
   margin: 0px 50px 0px 0px;
 }
 
+/*Post with 2 images side by side*/
```

Comment:
Hi @GiuliaMantuano, 
Can you please work with @neomatrix369 to resolve the conflicts?